brainglobe_atlasapi/atlas_generation/atlas_scripts/perens_stereotaxic_mri_mouse.py

```diff
@@ -0,0 +1,402 @@
+__version__ = "0"
+import json
+import multiprocessing as mp
+import os
+import time
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import py7zr
+import SimpleITK as sitk
+from rich.progress import track
+
+from brainglobe_atlasapi import utils
+from brainglobe_atlasapi.atlas_generation.mesh_utils import (
+    Region,
+    create_region_mesh,
+)
+from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+from brainglobe_atlasapi.structure_tree_util import get_structures_tree
+
+PARALLEL = True  # disable parallel mesh extraction for easier debugging
+
+
+### Additional functions #####################################################
```

Comment:
To avoid code duplication, I would suggest to use existing functions to map from id to acronym and other queries on the Allen atlas. Some example relevant functionality from an IPython notebook:
```
In [11]: atlas = BrainGlobeAtlas("allen_mouse_25um")

In [12]: atlas.structures.acronym_to_id_map["mp"]
Out[12]: 673

In [13]: atlas.structures["673"]
Out[13]: {'acronym': 'mp', 'id': 673, 'name': 'mammillary peduncle', 'structure_id_path': [997, 1009, 991, 824, 46, 673], 'rgb_triplet': [204, 204, 204], 'mesh_filename': PosixPath('/home/alessandro/.brainglobe/allen_mouse_25um_v1.2/meshes/673.obj'), 'mesh': None}

In [14]: atlas.structures["673"]["acronym"]
Out[14]: 'mp'

In [15]: atlas.get_structure_ancestors("673")
Out[15]: ['root', 'fiber tracts', 'mfbs', 'mfsbshy', 'mfbsma']
```

Reply:
I tried to address this in the latest commit, let me know if its what you had in mind.